<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="NickName_lbl">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>330</y>
     <width>230</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BM EULJIRO TTF</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color : #fff0cb;</string>
   </property>
   <property name="text">
    <string>NickName</string>
   </property>
  </widget>
  <widget class="QLabel" name="pwSignCnf_lbl">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>230</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BM EULJIRO TTF</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color : #fff0cb;</string>
   </property>
   <property name="text">
    <string>PW Confirm</string>
   </property>
  </widget>
  <widget class="QLabel" name="idSign_lbl">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BM EULJIRO TTF</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color : #fff0cb;</string>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="pwSign_lbl">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BM EULJIRO TTF</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color : #fff0cb;</string>
   </property>
   <property name="text">
    <string>PW</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idSign_lineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>235</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BM YEONSUNG</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="tabletTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;
padding: 10px;
</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="nick_lineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>360</y>
     <width>235</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BM YEONSUNG</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="tabletTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;
padding: 10px;
</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwSignCnf_lineEdit">
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>290</y>
     <width>235</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BM YEONSUNG</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="tabletTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;
padding: 10px;
</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwSign_lineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>235</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BM YEONSUNG</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="tabletTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;
padding: 10px;
</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="signUpBackground_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>500</height>
    </rect>
   </property>
   <property name="pixmap">
    <pixmap>../image/playScreen/signup_Screen.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_btn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>415</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>415</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>signUpBackground_label</zorder>
  <zorder>NickName_lbl</zorder>
  <zorder>pwSignCnf_lbl</zorder>
  <zorder>idSign_lbl</zorder>
  <zorder>pwSign_lbl</zorder>
  <zorder>idSign_lineEdit</zorder>
  <zorder>nick_lineEdit</zorder>
  <zorder>pwSignCnf_lineEdit</zorder>
  <zorder>pwSign_lineEdit</zorder>
  <zorder>ok_btn</zorder>
  <zorder>cancel_btn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
